hubspot/connection/__init__.py

```diff
@@ -204,6 +204,7 @@ def _require_successful_response(response):
             raise exception_class(
                 error_data['message'],
                 error_data['requestId'],
+                error_data,
```

Comment:
Hi @LuRsT, `failureMessages` is used when there are errors during a bulk contact update - it contains the specific error that happened for each contact that failed to be updated (and the index of it so that you can figure out which of the contacts that you sent the error applies to).

If you look in `test_client_error_response` then you can see an example of what `failureMessages` looks like. I think you can reproduce a similar error by using save_contacts (in hubspot-contacts) with a contact with no email address (or if that doesn't work then try a contact with the empty string as email address).

If you think that giving failureMessages special treatment isn't the right way to go then I can remove it - client code would still be able to get at the failureMessages by poking around in error_data.